federatedscope/core/regularizer/l2_regularizer.py

```diff
@@ -0,0 +1,36 @@
+from federatedscope.register import register_regularizer
+from torch.nn import Module
+
+import torch
+
+REGULARIZER_NAME = "l2_regularizer"
+
+
+class L2Regularizer(Module):
+    """Returns the l2 norm of weight
+
+        Arguments:
+            p (int): The order of norm.
```

Comment:
'p=2'?

Reply:
modified accordingly

Reply:
In the funcational mechanism original paper, the noise addition process is performed only once for the objective function, while your implementation performs noise addition in each iteration, is this supported by the theory?